pinot-core/src/test/java/org/apache/pinot/queries/TextSearchQueriesTest.java

```diff
@@ -345,7 +373,11 @@ public void testTextSearch()
         "SELECT INT_COL, SKILLS_TEXT_COL FROM MyTable WHERE TEXT_MATCH(SKILLS_TEXT_COL, '\"Machine learning\"') LIMIT 50000";
     testTextSearchSelectQueryHelper(query, expected.size(), false, expected);
 
-    query = "SELECT COUNT(*) FROM MyTable WHERE TEXT_MATCH(SKILLS_TEXT_COL, '\"Machine learning\"') LIMIT 50000";
+    query =
+        "SELECT INT_COL, SKILLS_TEXT_COL FROM MyTable WHERE SKILLS_TEXT_COL LIKE '\"Machine learning\"' LIMIT 50000";
+    testTextSearchSelectQueryHelper(query, expected.size(), false, expected);
+
+    query = "SELECT COUNT(*) FROM MyTable WHERE SKILLS_TEXT_COL LIKE '\"Machine learning\"' LIMIT 50000";
```

Comment:
Can you also add few more tests  -- essentially a LIKE version of majority of tests here to test out multiple phrases using boolean, regex, prefix etc. Right now the LIKE tests are only testing single phrases I guess

Reply:
Since LIKE essentially supports single term and "%" (* in regex) type queries, I have added tests for them.